tests/UserRepositoryTest.php

```diff
@@ -147,4 +133,19 @@ protected function putJSONFixture($data, $fixtureName)
     {
         return file_put_contents($fixtureName, json_encode($data, JSON_PRETTY_PRINT));
     }
-}
+
+    protected function assertExceptionThrowed($class, $callback, $message)
```

Comment:
let's change the order of arguments

```suggestion
    protected function assertExceptionThrowed($expectedExceptionClass, $expectedMessage, $callback)
```